src/viam/media/media.py

```diff
@@ -0,0 +1,48 @@
+from collections.abc import AsyncIterator
+from typing import Protocol, TypeVar
+
+MediaType = TypeVar("MediaType", covariant=True)
+
+
+class MediaStream(Protocol[MediaType]):
+    async def next(self) -> MediaType:
+        ...
```

Comment:
just for my own understanding: how does the `...` syntax differ from `Pass`?  And, would it be helpful to have an `unimplemented` message returned rather than not doing anything?

Reply:
No real difference between `...` and `pass`. Stylistically I prefer `...` because to me it's signals "implementation defined elsewhere," whereas pass feels more like a noop.